app/models/course.rb

```diff
@@ -263,10 +263,40 @@ def sites_not_associated_with_course
   end
 
   def has_bursary?
+    # https://bat-design-history.herokuapp.com/find-teacher-training/financial-incentives-multiple-subjects
+    # If course name contains "with"
+    return true if name.include?("with")
```

Comment:
I'm not convinced that this logic is right – I don't think the unit tests from the C# side will pass (see https://github.com/DFE-Digital/search-and-compare-ui/pull/335). In any case we should port those over.